contracts/PriceOracle/PriceOracle.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.7;
+
+import { Ownable } from "../Ownable.sol";
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import { FullMath } from "../Uniswap/UniswapLib.sol";
+
+contract PriceOracle is Ownable {
+
+    /// @dev Configuration used to return the USD price for the associated cToken asset and base unit needed for formatting
+    /// There should be 1 TokenConfig object for each supported asset, passed in the constructor.
+    struct TokenConfig {
+        // The address of the Compound Token
+        address cToken;
+        // The number of smallest units of measurement in a single whole unit. (e.g. 1e18 for ETH)
+        uint256 baseUnit;
```

Comment:
Pack struct to save gas.

https://dev.to/javier123454321/solidity-gas-optimizations-pt-3-packing-structs-23f4

```suggestion
    struct TokenConfig {
       uint256 baseUnit;
       address cToken;
       address priceFeed;
    }
   
```

Reply:
I had considered it originally but remember coming to the conclusion none of them to pack with the other so the order didn't matter. My reasoning being addresses are 20 bytes so they can't pack with each other. I'm likely missing something though

Reply:
Actually yes you are correct.  Ignore this.